README.md

```diff
@@ -1771,6 +1771,22 @@ end
 
 The `rescue_from` block must return a `Rack::Response` object, call `error!` or re-raise an exception.
 
+The `with` keyword is available as `rescue_from` options, it can be passed method name or `Rack::Response` object.
+
+```ruby
+class Twitter::API < Grape::API
+  format :json
+  helpers do
+    def server_error!
+      error!({ error: 'Server error.' }, 500, { 'Content-Type' => 'text/error' })
+    end
+  end
+
+  rescue_from :all,          with: :server_error!
+  rescue_from ArgumentError, with: Rack::Response.new('rescued with a method', 400)
```

Comment:
I think `with: :method` is very useful, however a rescue with a `Rack::Response` is probably dangerous, because the instance of the `Rack::Response` is constructed at load time. This tends to cause all kinds of wonderful behavior like memory leaks, people are reckless with them. I think the above should not be allowed, you should do a `-> () { Rack...}` when you want to return something like this.